crates/lib/src/cli.rs

```diff
@@ -295,6 +295,12 @@ pub(crate) enum InstallOpts {
 /// Subcommands which can be executed as part of a container build.
 #[derive(Debug, clap::Subcommand, PartialEq, Eq)]
 pub(crate) enum ContainerOpts {
+    /// Output JSON to stdout containing the container image metadata.
```

Comment:
Oops I left the comment in :(

Edit: oh wait nevermind this isn't where I moved the struct 🤦‍♂️ 

Ignore me, too late on Friday!